explainability-core/README.md

```diff
@@ -1,10 +1,46 @@
-# TrustyAI Explainability Core Library
+# TrustyAI Explainability Core
+The TrustyAI Core Libraru is a Java library for explainable and transparent AI, 
```

Comment:
Typo and small suggestion to avoid repeating "library"
```suggestion
The TrustyAI Core Library is written in Java for explainable and transparent AI, 
```